pkg/util/helm/convert.go

```diff
@@ -0,0 +1,189 @@
+package helm
+
+/*
+  This package can be used to convert an existing helm style project into a Framework and FrameworkVersion for
+  importing into KUDO
+
+*/
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	kudo "github.com/kudobuilder/kudo/pkg/apis/kudo/v1alpha1"
+
+	"github.com/helm/helm/pkg/chartutil"
+)
+
+// Import convers the chart at the provided folder into a Framework and FrameworkVersion
+func Import(folder string) (kudo.Framework, kudo.FrameworkVersion, error) {
+
+	frameworkVersion := kudo.FrameworkVersion{}
+	frameworkVersion.Kind = "FrameworkVersion"
+	frameworkVersion.APIVersion = kudo.SchemeGroupVersion.String()
+
+	framework, err := loadMetadata(folder)
+	if err != nil {
+		return framework, frameworkVersion, err
+	}
+
+	templates, err := loadTemplates(folder)
+	if err != nil {
+		return framework, frameworkVersion, err
+	}
+	//need to get Version from chart.yaml
+	frameworkVersion.Name = framework.Name + "-stable"
+	frameworkVersion.Spec.Templates = templates
+
+	//Create the single task
+	deployTask := kudo.TaskSpec{
+		Resources: make([]string, 0),
+	}
+	for k := range templates {
+		deployTask.Resources = append(deployTask.Resources, k)
+	}
+	frameworkVersion.Spec.Tasks = make(map[string]kudo.TaskSpec)
+	frameworkVersion.Spec.Tasks["deploy"] = deployTask
+
+	//create the single plan
+	deployPlan := kudo.Plan{
+		Strategy: kudo.Serial,
+		Phases: []kudo.Phase{kudo.Phase{
+			Name:     "deploy",
+			Strategy: kudo.Serial,
+			Steps: []kudo.Step{
+				kudo.Step{
+					Name:   "deploy",
+					Tasks:  []string{"deploy"},
+					Delete: false,
+				},
+			},
+		}},
+	}
+	frameworkVersion.Spec.Plans = make(map[string]kudo.Plan)
+	frameworkVersion.Spec.Plans["deploy"] = deployPlan
+
+	//parameters
+	params, err := loadParameters(folder)
+	if err != nil {
+		return framework, frameworkVersion, err
+	}
+	frameworkVersion.Spec.Parameters = params
+
+	frameworkVersion.Spec.Framework.Name = framework.Name
+
+	return framework, frameworkVersion, nil
+}
+
+func loadParameters(folder string) ([]kudo.Parameter, error) {
+	params := make([]kudo.Parameter, 0)
+	values, err := chartutil.ReadValuesFile(folder + "/values.yaml")
+	if err != nil {
+		return params, err
+	}
+	params, err = getParametersFromValues(values)
+	if err != nil {
+		return params, err
+	}
+	//all the parameters start with a period
+	for i := range params {
+		params[i].Name = ".Values." + params[i].Name
+	}
+	return params, err
+}
+
+func getParametersFromValues(value chartutil.Values) ([]kudo.Parameter, error) {
+	params := make([]kudo.Parameter, 0)
+	for k, v := range value {
+		s, ok := v.(string)
+		if ok {
+			params = append(params,
+				kudo.Parameter{
+					Name:        k, // ???
```

Comment:
??? is unclear

Reply:
Thanks for the reminder. This was to better understand how we want to handle parameter names.  Right now the templates from helm all use the syntax `.Values.some.thing` but in the `values.yaml` file, they're referenced as 
```yaml
some:
  thing: value
```

So we could have our parameters mirror the `values.yaml` and just be `some.thing`, or have them align with the templates and have them as `.Values.some.thing`.

Thoughts?